src/client/app/helpers/trusted.snippet.helper.ts

```diff
@@ -0,0 +1,26 @@
+class TrustedSnippet {
+    private _trustedSnippetsKey = 'trusted_snippets';
+
+    isSnippetTrusted(snippetId: string, gistId: string): boolean {
+        let trustedSnippets = JSON.parse(window.localStorage.getItem(this._trustedSnippetsKey));
```

Comment:
I usually use try/catch whenever I do a JSON.parse due to the fact it throws an exception when parsing the empty string.

Reply:
Added the `try/catch`